cartographer/cloud/internal/local_trajectory_uploader.cc

```diff
@@ -171,68 +118,6 @@ void LocalTrajectoryUploader::TranslateTrajectoryId(
   sensor_metadata->set_trajectory_id(cloud_trajectory_id);
 }
 
-void LocalTrajectoryUploader::ProcessFixedFramePoseDataMessage(
-    proto::AddFixedFramePoseDataRequest *data_request) {
-  if (!add_fixed_frame_pose_client_) {
-    add_fixed_frame_pose_client_ = make_unique<
-        async_grpc::Client<handlers::AddFixedFramePoseDataSignature>>(
-        client_channel_);
-  }
-  TranslateTrajectoryId(data_request->mutable_sensor_metadata());
-  CHECK(add_fixed_frame_pose_client_->Write(*data_request));
-}
-
-void LocalTrajectoryUploader::ProcessImuDataMessage(
-    proto::AddImuDataRequest *data_request) {
-  if (!add_imu_client_) {
-    add_imu_client_ =
-        make_unique<async_grpc::Client<handlers::AddImuDataSignature>>(
-            client_channel_);
-  }
-  TranslateTrajectoryId(data_request->mutable_sensor_metadata());
-  CHECK(add_imu_client_->Write(*data_request));
-}
-
-void LocalTrajectoryUploader::ProcessOdometryDataMessage(
-    proto::AddOdometryDataRequest *data_request) {
-  if (!add_odometry_client_) {
-    add_odometry_client_ =
-        make_unique<async_grpc::Client<handlers::AddOdometryDataSignature>>(
-            client_channel_);
-  }
-  TranslateTrajectoryId(data_request->mutable_sensor_metadata());
-  CHECK(add_odometry_client_->Write(*data_request));
-}
-
-void LocalTrajectoryUploader::ProcessLandmarkDataMessage(
-    proto::AddLandmarkDataRequest *data_request) {
-  if (!add_landmark_client_) {
-    add_landmark_client_ =
-        make_unique<async_grpc::Client<handlers::AddLandmarkDataSignature>>(
-            client_channel_);
-  }
-  TranslateTrajectoryId(data_request->mutable_sensor_metadata());
-  CHECK(add_landmark_client_->Write(*data_request));
-}
-
-void LocalTrajectoryUploader::ProcessLocalSlamResultDataMessage(
-    proto::AddLocalSlamResultDataRequest *data_request) {
-  if (!add_local_slam_result_client_) {
-    add_local_slam_result_client_ = make_unique<
-        async_grpc::Client<handlers::AddLocalSlamResultDataSignature>>(
-        client_channel_);
-  }
-  TranslateTrajectoryId(data_request->mutable_sensor_metadata());
-  // A submap also holds a trajectory id that must be translated to uplink's
-  // trajectory id.
-  for (mapping::proto::Submap &mutable_submap :
-       *data_request->mutable_local_slam_result_data()->mutable_submaps()) {
-    mutable_submap.mutable_submap_id()->set_trajectory_id(
```

Comment:
Why is this no longer needed?

Reply:
Dude, you are awesome. Thanks for catching that!